Torquebox image

jhadvig opened the pull request:
Torquebox image for running Rack base apps.

jhadvig committed:
Torquebox image

vbalazs commented on centos-torquebox/Dockerfile:
```diff
@@ -0,0 +1,33 @@
+RUN mkdir -p /opt/torquebox && useradd torquebox -c"Torquebox system user" -M -ptorquebox
+
+RUN yum update --assumeyes && \
+	yum install --assumeyes java-1.7.0-openjdk.x86_64 wget unzip sqlite-devel libsqlite3x-devel nodejs010-nodejs zlib-devel libxslt-devel libxml2-devel && \
```
hi, are you sure you need nodejs?

jhadvig replied:
Thanks for noticing that, I've overlooked that one

vbalazs commented:
Sorry @jhadvig , it seems I was wrong. Apparently we need v8 for rake assets compilation therefore we need nodejs as a dependency.

See: https://github.com/openshift/ruby-19-centos/pull/1#discussion_r12703385
We also talked about it on the #openshift-dev channel (but I can't find the channel logs publicly)

Excuse me for my captiousness.

jhadvig commented:
@vbalazs 
No worries, found out that I'm not precompiling the assets anyway (for now).
Will update it :)
Thx

smarterclayton commented:
Is this updated?

jhadvig commented:
@smarterclayton  No, have to update it so it will work with latest sti. Will do that and will notice you.

jhadvig commented:
@smarterclayton I've updated the image. This image is not STI related, sorry for the comment above. Also added readme in case anyone wanted to try it.

smarterclayton commented on centos-torquebox/bin/run:
```diff
@@ -0,0 +1,13 @@
+
+torquebox deploy $app_root_dir --env=$RAILS_ENV
+
+torquebox run -b 0.0.0.0
```
use an exec here

jhadvig commented:
@smarterclayton updated. Thanks for review :)
